{
  "status": "SUCCESS",
  "cpu_usage": [
    "27"
  ],
  "stdout": [
    "EPI(k=10,i=  50): cost -6.51E+00, H  7.02E+00, |R|^2 2.64E-02\n",
    "EPI(k=10,i= 100): cost -6.92E+00, H  7.02E+00, |R|^2 1.48E-02\n",
    "EPI(k=10,i= 150): cost -6.87E+00, H  7.12E+00, |R|^2 1.18E-02\n",
    "EPI(k=10,i= 200): cost -6.76E+00, H  7.20E+00, |R|^2 2.09E-02\n",
    "EPI(k=10,i= 250): cost -7.04E+00, H  7.13E+00, |R|^2 8.31E-03\n",
    "EPI(k=10,i= 300): cost -6.86E+00, H  7.03E+00, |R|^2 1.25E-02\n",
    "EPI(k=10,i= 350): cost -7.16E+00, H  7.19E+00, |R|^2 4.78E-03\n",
    "EPI(k=10,i= 400): cost -7.15E+00, H  7.14E+00, |R|^2 1.03E-02\n",
    "EPI(k=10,i= 450): cost -6.94E+00, H  7.11E+00, |R|^2 8.15E-03\n",
    "EPI(k=10,i= 500): cost -6.92E+00, H  7.09E+00, |R|^2 8.40E-03\n",
    "EPI(k=10,i= 550): cost -7.00E+00, H  7.11E+00, |R|^2 2.03E-03\n",
    "EPI(k=10,i= 600): cost -6.90E+00, H  7.13E+00, |R|^2 9.79E-03\n",
    "EPI(k=10,i= 650): cost -7.00E+00, H  7.20E+00, |R|^2 5.11E-03\n",
    "EPI(k=10,i= 700): cost -6.85E+00, H  7.26E+00, |R|^2 2.32E-02\n",
    "EPI(k=10,i= 750): cost -6.92E+00, H  7.17E+00, |R|^2 1.78E-02\n",
    "EPI(k=10,i= 800): cost -6.93E+00, H  7.10E+00, |R|^2 4.19E-03\n",
    "EPI(k=10,i= 850): cost -7.13E+00, H  7.07E+00, |R|^2 3.25E-03\n",
    "EPI(k=10,i= 900): cost -7.18E+00, H  7.10E+00, |R|^2 2.09E-02\n",
    "EPI(k=10,i= 950): cost -7.09E+00, H  7.18E+00, |R|^2 3.71E-03\n",
    "EPI(k=10,i=1000): cost -6.96E+00, H  7.13E+00, |R|^2 6.42E-03\n"
  ],
  "instance": "i-01a66ff56bea6c74d",
  "reason": [
    "aws s3 sync ./ \"s3://$bucketname/$groupdir/$resultdir/per_hp\""
  ],
  "command": "6ec85979-22dd-4e33-bf61-ee5033b219c6",
  "stderr": [
    "2020-03-10 14:02:12.502533: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer.so.6'; dlerror: libnvinfer.so.6: cannot open shared object file: No such file or directory\n",
    "2020-03-10 14:02:12.502622: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer_plugin.so.6'; dlerror: libnvinfer_plugin.so.6: cannot open shared object file: No such file or directory\n",
    "2020-03-10 14:02:12.502632: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:30] Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.\n",
    "2020-03-10 14:02:47.124787: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1\n",
    "2020-03-10 14:02:47.586280: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected\n",
    "2020-03-10 14:02:47.586326: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (ip-172-31-71-67): /proc/driver/nvidia/version does not exist\n",
    "2020-03-10 14:02:47.586864: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA\n",
    "2020-03-10 14:02:47.709041: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2499995000 Hz\n",
    "2020-03-10 14:02:47.710074: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x56421e5ca420 initialized for platform Host (this does not guarantee that XLA will be used). Devices:\n",
    "2020-03-10 14:02:47.710104: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version\n",
    "WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/epi/lib/python3.6/site-packages/tensorflow_probability/python/bijectors/real_nvp.py:386: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.\n",
    "Instructions for updating:\n",
    "Use keras.layers.Dense instead.\n",
    "WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/epi/lib/python3.6/site-packages/tensorflow_core/python/layers/core.py:187: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.\n",
    "Instructions for updating:\n",
    "Please use `layer.__call__` method instead.\n",
    "WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/epi/lib/python3.6/site-packages/tensorflow_probability/python/bijectors/real_nvp.py:222: AffineScalar.__init__ (from tensorflow_probability.python.bijectors.affine_scalar) is deprecated and will be removed after 2020-01-01.\n",
    "Instructions for updating:\n",
    "`AffineScalar` bijector is deprecated; please use `tfb.Shift(loc)(tfb.Scale(...))` instead.\n"
  ],
  "input": "cunninghamlabEPI/inputs/epidata/dataset3.json"
}
